frontend/src/components/dashboard/campaigns/Campaigns.tsx

```diff
@@ -63,18 +66,45 @@ const Campaigns = () => {
     setLoading(false)
   }
 
+  // Returns true if A < B, else false
+  function compareSemver(A: string, B: string) {
```

Comment:
Minor nitpick. Generally we use camelCase for variable names. Perhaps more informative variable names `lastSeenVersion` and `currentVersion` also?